src/pe_reports/metrics.py

```diff
@@ -0,0 +1,37 @@
+"""Class methods for report metrics."""
+
+# cisagov Libraries
+from pe_reports.data.db_query import query_cyberSix_creds, query_hibp_view
+
+
+class Credentials:
+    """Credentials class."""
+
+    def __init__(self, start_date, end_date, org_uid):
+        """Initialize credentials class."""
+        self.start_date = start_date
+        self.end_date = end_date
+        self.org_uid = org_uid
+        self.query_cyberSix_creds = query_cyberSix_creds(org_uid, start_date, end_date)
+        self.query_hibp_view = query_hibp_view(org_uid, start_date, end_date)
+
+    def total(self):
+        """Return total number of credentials."""
+        df_cred_csg = self.query_cyberSix_creds.shape[0]
+        df_cred_hibp = self.query_hibp_view.shape[0]
+        total = df_cred_csg + df_cred_hibp
+        return total
```

Comment:
I'd get rid of this unnecessary `total` variable:
```suggestion
        return df_cred_csg + df_cred_hibp
```

Reply:
I addressed your recommendation per commit [f0fb31e](https://github.com/cisagov/pe-reports/pull/90/commits/f0fb31e5b4050af3f57a8b5bf5228f963de58df8)